src/main/java/com/google/cloud/tools/maven/RunMojo.java

```diff
@@ -214,6 +214,28 @@ protected void workAroundNonJava7Version() {
     }
   }
 
+  /**
+   * Determine if the application is a Standard Environment app.
+   */
+  protected boolean isStandardEnvironmentApp() {
+    return mavenProject != null
+        && mavenProject.getBuild() != null
+        && new File(
+        mavenProject.getBuild().getDirectory() + "/" + mavenProject.getBuild().getFinalName()
```

Comment:
Since, we're running the Dev Appserver from the build dir, I thought it would be more appropriate to check there, rather than in the source. I'll modify the comment to make it more clear.